app/src/test/kotlin/com/wire/android/ui/home/conversations/call/ConversationListCallViewModelTest.kt

```diff
@@ -110,7 +114,8 @@ class ConversationListCallViewModelTest {
             observeConversationDetails = observeConversationDetails,
             observeParticipantsForConversation = observeParticipantsForConversation,
             setUserInformedAboutVerification = setUserInformedAboutVerificationUseCase,
-            observeDegradedConversationNotified = observeDegradedConversationNotifiedUseCase
+            observeDegradedConversationNotified = observeDegradedConversationNotifiedUseCase,
+            getSelf = getSelfUserUseCase
```

Comment:
Can we write tests for it?

Reply:
Tests added and migrated to arrangements